cloud-provider-vsphere.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: cloud-provider-vsphere
+  version: "1.33.0"
+  epoch: 0
+  description: Kubernetes Cloud Provider for vSphere
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
```

Comment:
Did you inspect the upstream image for signs of any additional image config that we need to handle here? Pretty sure you'll need a compat package at a minimum